spec/02-integration/09-hybrid_mode/11-status_spec.lua

```diff
@@ -119,7 +116,10 @@ for _, strategy in helpers.each_strategy() do
 
           local status = res and res.status
           http_client:close()
-          if status == 200 then
+
+          if (inc_sync == "on" and status == 503) or
```

Comment:
It could work but may not the best solution, since the case title is `"should return 200 on data plane after configuring"`, and the status code is `503` when enabling inc_sync.